module-2/backend/week_three.md

```diff
@@ -10,14 +10,42 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new <project name> –database=postgresql 
+
 2. What do Models generally inherit from in rails?
+ActiveRecord::Base
+
 3. What do Controllers generally inherit from in a rails project?
+ApplicationController
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+resources :horses, :only => [:show]
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+Rake routes gives us a list of all available routes including the path, path helper, and controller method.
+
 6. What is an example of a route helper? When would you use them?
+An example would be horses_path. It’s the helper for the /horses path and controller#index view.
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+The url would provide the domain name, like www.domain_name.com/users/horses. The path here could be users_horses_path.
+
 8. What are strong params and why are the necessary?
+They are a collection of params that are allowed to pass to our model. We use them to control what params can go to the model without any more processing from the controller.
+
 9. What role does `form_for` play in helping us create our forms?
+Form_for helps us set up the html side of the form, including labels, boxes, and submit buttons.
+
 10. How does `form_for` know where to submit the user's input?
+It knows based on the input variable or variables placed within the form.
```

Comment:
I think yes? Assuming by 'input variable' you mean the 'argument', e.g. `@horse` in the example below.